content/4_Gold/Intro_Bitwise.mdx

```diff
@@ -28,10 +29,102 @@ redirects:
 	</Resource>
 </Resources>
 
-Let's take a look at some examples to better understand bitwise operators. Note
-that you typically won't be asked to use bitwise operators to implement
-addition/multiplication.
+At this point, you should already be familiar with the three main bitwise operators (AND, OR, and XOR).
+Let's take a look at some examples to better understand them.
+
+<FocusProblem problem="sample" />
+
+## Solution - Take a Guess
+
+<Spoiler title="Hint">
+
+Our queries are quite limited, as we can only find the AND and OR of two numbers. Maybe we can
+combine the usage of these two operations to find the value of an operation that would be more
+useful to us?
+
+</Spoiler>
+
+<Spoiler title="Solution">
+
+In fact, we can figure out the sum of two numbers using just their AND, OR and XOR values! Suppose we know their
+XOR values, we can use the following property:
+
+$a + b = 2 \cdot (a \& b) + a \oplus b$
+
+<Spoiler title="Proof">
```

Comment:
why spoiler the proof

Reply:
idk maybe that want to do it for themselves first

Reply:
add some text in front of it "maybe you wanna do it for yourself, but the proof is here"

Reply:
it looks ugly, but if you insist i'll do it

Reply:
did you do it

Reply:
your mother